src/meshapi/templates/widgets/uisp_link.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n static %}
+<div class="related-widget-wrapper">
+    {% include "django/forms/widgets/input.html" %}
+    {% block links %}
+    {% spaceless %}
+    {% if widget.value %}
+    <a class="related-widget-wrapper-link view-related"
+            href="https://uisp.mesh.nycmesh.net/nms/devices#id={{ widget.value }}&panelType=device-panel"
```

Comment:
Is it worth the effort to make this configurable? If anyone feels strongly about it, I'll just make it default to this.